docs/pages/x/api/data-grid/data-grid-pro.json

```diff
@@ -1375,6 +1369,30 @@
       "description": "Styles applied to the horizontal scrollbar.",
       "isGlobal": false
     },
+    {
+      "key": "scrollbarFiller",
```

Comment:
Should we ignore the scrollbar filler classes?

Reply:
Wdym by ignore? I already have the `@ignore - do not document` on there, but the export script still picks these ones up.

Reply:
I've merged `next` into the branch - they're properly ignored now